testhttp/__init__.py

```diff
@@ -192,15 +193,19 @@ def run_tests(self):
 
 
 class HTTPProcessor:
-    def __init__(self, files):
+    def __init__(self, files, vars):
         self.http_opjects = []
         self.http_objects_by_name = {}
         self.vars = {}
         self.success = 0
         self.failures = 0
         self.cwd = None
 
-        for file in files.split(','):
+        for variable in [variable for variable in (vars or [])]:
```

Comment:
Can you explain the reason why you did the list comprehension here instead of just in (vars or [])?

Reply:
No special reason. 
Changed it to 'in (vars or [])'